prelude/align.md

```diff
@@ -3,7 +3,7 @@ See [Align](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_target/abi/str
 
 `Align` is always a power of two.
 
-```rust
+```rust,ignore
```

Comment:
Can you also remove the method bodies (for all `ignore` blocks)?

Reply:
@RalfJung alright, done!